### DISCUSSION TITLE
address_1 textarea

### DISCUSSION THREAD

**Discussion by aunco** [woocommerce/woocommerce#52163] [Checkout Flow]
Please change address_1 to textarea. In the country of Malaysia, newer addresses are typically very long.

**Comment by mikejolley** [comment]
Hi

I think its unlikely we'd change the default from text to text area for one location, but can you provide an example? Is it not broken into the other fields rather than one long field?

I can find no examples of long addresses on wiki etc so any real world examples would be useful.

Thanks
